cetmix_tower_server/models/cx_tower_server.py

```
@@ -640,6 +640,50 @@ def action_uninstall_jet_template(self):
+                for omitting ``host_key``. Defaults to False.
+
+        Returns:
+            dict: Keys ``host``, ``port``, ``username``, ``auth_mode``,
+                ``password``, ``ssh_key``, ``host_key``, ``skip_host_key``.
+                ``host_key`` is None when skip is in effect.
+
+        Raises:
+            ValidationError: If a host key is required and missing.
+        """
+        self.ensure_one()
+        self = self.sudo()
```

Ruff (0.16.2)

[warning] Reassigned `self` variable in instance method

(PLW0642)